src/Schedule.h

```diff
@@ -441,6 +441,10 @@ struct Dim {
      * loop (see the DimType enum above). */
     DimType dim_type;
 
+    /** Allows whether the PartitionLoops lowering pass is allowed
```

Comment:
Any additions here also require updates to serialization and deserialization code. I suggest just copying how some other field is handled (i.e. dim_type).